### ISSUE TITLE
exclude option not working with latest babel 7 stable release

### ISSUE THREAD

**Issue by micabe** [MatthieuLemoine/unused-webpack-plugin#14]
2.2.0 does not match master branch

**Comment by AlahmadiQ8** [comment]
I just confirmed that the npm version is different than the current version on github. 

i don't think the issue is related to babel 7 but has to do with this PR https://github.com/MatthieuLemoine/unused-webpack-plugin/pull/12 where files in nested folders are not caught by the exclude patter and you must add `**/*.test.js` instead of simply `*.test.js`. 

However, this is already fixed with `deglob` library introduced in PR https://github.com/MatthieuLemoine/unused-webpack-plugin/pull/13 which works correctly with exclude patters. 

That said, PR https://github.com/MatthieuLemoine/unused-webpack-plugin/pull/13 is still not on npm. 

**tldr** simply publishing the latest version to npm should solve the issue

**Comment by micabe** [comment]
Yep, thanks @AlahmadiQ8 for the given explanation. I found out about the deglob lib.
Waiting for new publication

**Comment by MatthieuLemoine** [comment]
[v2.3.0](https://github.com/MatthieuLemoine/unused-webpack-plugin/releases/tag/v2.3.0) published
